style.css

```diff
@@ -1,3 +1,78 @@
 * {
   box-sizing: border-box;
 }
+
+.minDiv {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.minDiv h1 {
+  margin-bottom: 5px;
+}
+
+.hrDiv {
+  width: 20%;
+  margin-top: 30px;
+}
+
+.hrDiv hr {
+  height: 5px;
+  background-color: black;
+  border-radius: 30px;
+}
+
+.minDiv h2 {
+  margin-bottom: 5px;
+}
+
+.tableContainer {
+  border: 1px solid rgb(7, 106, 103);
+  width: 80%;
+}
+
+.tableRow {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.tableRow button {
+  border: 2px solid black;
+  margin-right: 8px;
+  box-shadow: 2px 2px black;
+  text-shadow: 0.5px 0.5px black;
+  text-decoration: solid;
+}
+
+#bookList {
+  border: 1px solid black;
+  width: 100%;
+}
+
+#bookList tr:nth-child(odd) {
+  background-color: #d5d1d1;
+}
+
+#addNewForm {
+  display: flex;
+  flex-direction: column;
+  align-items: end;
+  margin: 0;
+  height: 30px;
+}
+
+#addNewForm input {
+  border: 2px solid black;
+  margin-bottom: -12px;
+}
```

Comment:
- [ ] This is close as well but the book name input field is overlapping with the author's input field and it's causing a discrepancy with the given wireframe.
- Your design:
  ![firefox_FCda0jXlEN](https://user-images.githubusercontent.com/104968236/234267960-7cfb1748-fe3f-4ade-8138-d96800ea9de4.png)
- I believe the problem is being caused by the `margin-bottom: -12px;` property at line 69 so please try to fix it. ✔

Reply:
@AbelG101 
resolved